src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -139,7 +139,101 @@ def list_dsn_members(self, dataset_name, member_pattern=None,
         custom_args["headers"]["X-IBM-Attributes"] = attributes
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json['items']  # type: ignore
-
+    
+    def copy_uss_to_dataset(self,from_filname,to_dataset_name,to_member_name="",type="text",replace=False):
+        """
+        Copy a USS file to dataset 
+            Parameters
+            ----------
+            from_filname: str
+                Name of the file to copy from
+            to_dataset_name: str
+                Name of the dataset to copy to
+            to_member_name: str
+                Name of the member to copy to  
+            type: str
+                Type of the file to copy from
+            enq: str
+                Enqueue type for the file to copy from
+            replace: bool
+                If true, the function uses the REPLACE=YES on ARCHDEF request, otherwise it uses the REPLACE=NO.
```

Comment:
Not everyone will be familiar with what an ARCHDEF request is. For the replace option I think we'll want to say something like the following:
`If true, members in the target data set are replaced.`